sdk/eventhub/azure-eventhub/stress/templates/network_loss.yaml

```diff
@@ -1,31 +1,31 @@
-apiVersion: chaos-mesh.org/v1alpha1
-kind: NetworkChaos
-metadata:
-  name: '{{ .Release.Name }}-{{ .Release.Revision }}'
-  namespace: {{ .Release.Namespace }}
-  annotations:
-    experiment.chaos-mesh.org/pause: 'true'
-  labels:
-    scenario: 'stress'
-    release: '{{ .Release.Name }}'
-    revision: '{{ .Release.Revision }}'
-spec:
-  scheduler:
-    cron: '@every 30s'
-  duration: '10s'
-  action: loss
-  direction: to
-  externalTargets:
-    - 'servicebus.windows.net'
-  mode: one
-  selector:
-    labelSelectors:
-      testInstance: "eventhub-{{ .Release.Name }}-{{ .Release.Revision }}"
-      chaos: 'true'
-    namespaces:
-      - {{ .Release.Namespace }}
-    podPhaseSelectors:
-      - 'Running'
-  loss:
-    loss: '100'
-    correlation: '100'
+# apiVersion: chaos-mesh.org/v1alpha1
```

Comment:
This entire file is commented out. Do we need it or is it used for some other tests?

Reply:
I am not exactly sure but at some point I think this was used to create network chaos, it caused failures when deploying so I had to comment it out. I can make an issue/ address this file in another PR with the ReadME being added/ additional testing scenarios.

Reply:
yeah create an issue so we can look at it later if we need network chaos in our tests

Reply:
Happy to help with the failures. I would bet it's because the examples you were pulling from coincided with our schema migration to the new chaos mesh versions.